push-athena-data-to-rds/push-athena-data-to-rds.go

```diff
@@ -142,8 +143,10 @@ func checkVariablesPresence() bool {
 	_, isPrimaryDomainTable := os.LookupEnv("PRIMARY_DOMAIN_TABLE")
 	_, isVarnishTable := os.LookupEnv("VARNISH_TABLE")
 	_, isHaproxyTable := os.LookupEnv("HAPROXY_TABLE")
+	_, isGumletDB := os.LookupEnv("GUMLET_DB")
+	_, isGumletTable := os.LookupEnv("GUMLET_TABLE")
 
-	if isS3PathPresent && isBucketNamePresent && isBadgerHost && isBadgerAuth && isCloudflareDB && isAlbDB && isVarnishTable && isAssettypeTable && isPrimaryDomainTable && isHaproxyTable {
+	if isS3PathPresent && isBucketNamePresent && isBadgerHost && isBadgerAuth && isCloudflareDB && isAlbDB && isVarnishTable && isAssettypeTable && isPrimaryDomainTable && isHaproxyTable && isGumletDB && isGumletTable {
```

Comment:
what is this?

Reply:
I am checking if all the required environment variable are present.